src/routes/RequestsRoute.js

```diff
@@ -212,6 +219,12 @@ class RequestsRoute extends React.Component {
       staffSlips: PropTypes.shape({
         GET: PropTypes.func,
       }),
+      pickSlips: PropTypes.shape({
+        GET: PropTypes.func,
+      }),
```

Comment:
I think we can be sure that this one is required as well as the presence of the GET method, right? If so please add these for _pickSlips_ and _currentServicePoint_.